### PR TITLE
Add Grid.Blazor component

### PR DESCRIPTION
**Pull request by gustavnavar**
Please consider adding Grid.Blazor to the Libraries section.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [ChartJs.Blazor](https://github.com/mariusmuntean/ChartJs.Blazor) &ndash; Blazor Component that wraps ChartJS.
 * [DevExpress Blazor UI Components](https://github.com/DevExpress/RazorComponents) &ndash; UI components including Data Grid, Pivot Grid, and several data editors.
 * [Elmah.Io.Blazor](https://github.com/elmahio/Elmah.Io.Blazor) &ndash; Logs from Blazor to [elmah.io](https://elmah.io/) using Microsoft.Extensions.Logging.
+* [Grid.Blazor](https://github.com/gustavnavar/Grid.Blazor) &ndash; Grid component for Blazor and Mvc, supporting filtering, sorting and paging.
```
Comment (guardrex) [inline community.md]:
```suggestion
* [Grid.Blazor](https://github.com/gustavnavar/Grid.Blazor) &ndash; Grid component for Blazor and MVC, supporting filtering, sorting, and paging.
```

**Comment by guardrex** [comment]
@gustavnavar I can't edit a private branch. Please make the suggested change on your branch :point_up:. I'll call for review right after that.

**Comment by gustavnavar** [comment]
I think I deleted my private branch by mistake.
I will close this pull request and create a new one

**Comment by guardrex** [comment]
Sure thing ... use that suggested text :point_up:.

**Pull request closed without merging**